compiler_test.go

```diff
@@ -0,0 +1,26 @@
+package jsonschema
+
+import "testing"
+
+func TestCompiler_AddResourceJSON(t *testing.T) {
```

Comment:
since `AddResource` now calls `AddResourceJSON`, this test is no longer necessary. please delete this test file

Reply:
applied